style-guide: refresh page: add `{{string}}` placeholder (#9647)
tldr-pages/tldr: contributing-guides/style-guide.md

  and use the `{{/dev/sdXY}}` placeholder for *block devices* instead of `/dev/sda1`.
- If a command can take a variable number of arguments, use an ellipsis: `{{arg1 arg2 ...}}`.
  If one of the multiple options is possible, write it as `{{either|or}}`.

> - If a placeholder accepts any string value as input, then use the generic placeholder `{{string}}` without duplicating the option name (`--message "{{string}}"` instead of `--message "{{message}}").

Reviewer: I think this is overly specific and should be decided on a case-by-case basis.

  and use the `{{/dev/sdXY}}` placeholder for *block devices* instead of `/dev/sda1`.
- If a command can take a variable number of arguments, use an ellipsis: `{{arg1 arg2 ...}}`.
  If one of the multiple options is possible, write it as `{{either|or}}`.

> - If a placeholder can take any string value then use `{{any}}` generic placeholder
>   not to duplicate option name: `--message "{{any}}"` instead of
>   `--message "{{message}}"`.

Reviewer 2: ```suggestion
- If a placeholder can take any string value then use the `{{string}}` generic placeholder
  not to duplicate the option name, `--message "{{string}}"` instead of
  `--message "{{message}}"`.
```
How about this?

Author: I like it: it shows that string is expected. :)

> - If a placeholder can take any string value then use the `{{string}}` generic placeholder
>   not to duplicate the option name, `--message "{{string}}"` instead of
>   `--message "{{message}}"`.

Reviewer 2: ```suggestion
- If a placeholder accepts any string value as input, then use the generic placeholder `{{string}}` without duplicating the option name (`--message "{{string}}"` instead of `--message "{{message}}").
```
